site/docs/sql/intro.md

```diff
@@ -49,14 +49,21 @@ data types:
 ```sql
 CREATE MAPPING trades (
```

Comment:
Extend the sample so that it includes Kafka key as well. People use keys in their Kafka topics.